app/views/RoomView/RightButtons.tsx

```
@@ -487,7 +493,7 @@ class RightButtonsContainer extends Component<IRightButtonsProps, IRigthButtonsS
 			if (!this.isOmnichannelPreview()) {
 				return (
 					<HeaderButton.Container>
-						<HeaderButton.Item iconName='kebab' onPress={this.showMoreActions} testID='room-view-header-omnichannel-kebab' />
+						<HeaderButton.Item iconName='kebab' onPress={this.showMoreActions} testID='room-view-header-omnichannel-kebab' disabled={isVoipCallActive} />
```

ESLint

[error] Replace `·iconName='kebab'·onPress={this.showMoreActions}·testID='room-view-header-omnichannel-kebab'·disabled={isVoipCallActive}·` with `⏎↹↹↹↹↹↹↹iconName='kebab'⏎↹↹↹↹↹↹↹onPress={this.showMoreActions}⏎↹↹↹↹↹↹↹testID='room-view-header-omnichannel-kebab'⏎↹↹↹↹↹↹↹disabled={isVoipCallActive}⏎↹↹↹↹↹↹`

(prettier/prettier)